src/component/variantPage/TherapeuticImplication.tsx

```diff
@@ -1,220 +1,43 @@
+import _ from 'lodash';
 import * as React from 'react';
 import { observer } from 'mobx-react';
-import _ from 'lodash';
-// TODO remove the IndicatorQueryTreatment import after exposed in cbioportal-frontend-commons
-import {
-    IndicatorQueryResp,
-    IndicatorQueryTreatment,
-} from 'oncokb-ts-api-client';
-import { DefaultTooltip } from 'cbioportal-frontend-commons';
-import { generateOncokbLink, ONCOKB_URL } from './biologicalFunction/Oncokb';
 
-import functionalGroupsStyle from './functionalGroups.module.scss';
+import { IndicatorQueryResp } from 'oncokb-ts-api-client';
+import { ICivicVariantIndex, ICivicVariantSummary } from 'cbioportal-utils';
+
+import OncoKb from './therapeuticImplication/OncoKb';
+import Civic from './therapeuticImplication/Civic';
 
 interface ITherapeuticImplicationProps {
     oncokb: IndicatorQueryResp | undefined;
+    civic?: ICivicVariantIndex;
     isCanonicalTranscriptSelected: boolean;
 }
 
-export const SENSITIVE_LEVELS = [
```

Comment:
added a new component under `src/component/variantPage/therapeuticImplication/OncoKb.tsx` and moved OncoKB related code there.